openpype/modules/slack/plugins/publish/integrate_slack_api.py

```diff
@@ -377,8 +381,12 @@ def send_message(self, channel, message, publish_files):
             return response.data["ts"], file_ids
         except SlackApiError as e:
             # # You will get a SlackApiError if "ok" is False
-            error_str = self._enrich_error(str(e.response["error"]), channel)
-            self.log.warning("Error happened {}".format(error_str))
+            if e.response.get("errror"):
```

Comment:
I suspect this should be `error`?
```suggestion
            if e.response.get("error"):
```